src/Molarity.elm

```diff
@@ -41,6 +42,28 @@ type alias Molarity =
     Quantity Float MolesPerCubicMeter
 
 
+
+---------- CONSTANTS ----------
+
+
+{-| One mole per liter, in moles per cubic meter
+-}
+oneMolePerLiter : Float
+oneMolePerLiter =
+    Constants.mole / Constants.liter
```

Comment:
Nice! Okay, this is what I was puzzling about how to do this.

Reply:
Yeah, still kind of figuring out the right way to do this 🙂

Reply:
Yeah, still kind of figuring out the right way to do this 🙂